B/Blosc2/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Blosc2"
+version = v"1.9.9"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Blosc/c-blosc2.git", "f0a05c9df216d156104756ffbbdf7ba8e553eb2a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd c-blosc2/
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=OFF -DBUILD_BENCHMARKS=OFF -DBUILD_EXAMPLES=OFF -DBUILD_STATIC=OFF ..
+make -j${nproc}
+make install
+install_license ../LICENSES/*.txt
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libblosc2", :libblosc2)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

Comment:
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", lock_microarchitecture=false)
```
should fix the issue on ARM platforms.  It won't work on all ARM chips because they force specific extensions though.

Note that this requires Julia v1.6, is it ok for you?  If so, we can also build for additional platforms including `aarch64-apple-darwin` (also known as Apple Silicon)

Reply:
>Note that this requires Julia v1.6, is it ok for you? If so, we can also build for additional platforms including aarch64-apple-darwin (also known as Apple Silicon)